.config/forest.dic

```diff
@@ -17,3 +17,8 @@ tuple
 mutex
 RPC
 parameterize/D
+P2P
+synchronizer
+nonces
+async
+incentivize
```

Comment:
I'm kind of puzzled why do we need to add this. It's a normal word, no? Perhaps we can use a better "general" dictionary? Same goes for `nonce`.

Reply:
Yeah, the builtin dictionary only has 48k words. My up-to-date local dictionary has 79k words (including `nonce` and `incentivize`). The dictionary is 841KiBi. Probably worth it to include it in the repository.

Reply:
Let's do it! :muscle: